flow/layers/backdrop_filter_layer_unittests.cc

```diff
@@ -147,17 +150,19 @@ TEST_F(BackdropFilterLayerTest, Nested) {
   auto mock_layer2 = std::make_shared<MockLayer>(child_path2, child_paint2);
   auto layer1 = std::make_shared<BackdropFilterLayer>(layer_filter1);
   auto layer2 = std::make_shared<BackdropFilterLayer>(layer_filter2);
+  auto clip_layer = std::make_shared<ClipRectLayer>(clip_rect, Clip::hardEdge);
   layer2->Add(mock_layer2);
   layer1->Add(mock_layer1);
-  layer1->Add(layer2);
+  layer1->Add(clip_layer);
+  clip_layer->Add(layer2);
 
   SkRect children_bounds = child_path1.getBounds();
```

Comment:
Check for unused variables. In this case, I'm not sure the compiler might be able to realize that this is unused because the variable is involved in a join call after this which we can see is unimportant, but the compiler may not know that. We have similar variables displaced in the other tests modified by these changes that might become unused as well.